src/Tasks/AssemblyDependency/ResolveAssemblyReference.cs

```diff
@@ -2242,16 +2258,25 @@ out _copyLocalFiles
                     }
 
                     bool useSystemRuntime = false;
+                    bool useNetStandard = false;
                     foreach (var reference in dependencyTable.References.Keys)
                     {
                         if (string.Equals(SystemRuntimeAssemblyName, reference.Name, StringComparison.OrdinalIgnoreCase))
                         {
                             useSystemRuntime = true;
+                        }
+                        if (string.Equals(NETStandardAssemblyName, reference.Name, StringComparison.OrdinalIgnoreCase))
+                        {
+                            useNetStandard = true;
+                        }
+                        if (useSystemRuntime && useNetStandard)
```

Comment:
`should this be `useSystemRuntime || useNetStandard`

Reply:
@weshaggard No, we only want to break out of the loop if we've found both of them.  Otherwise we won't properly detect the case where both System.Runtime and netstandard are in the dependency closure.